### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by briaryoung** [Monika-After-Story/MonikaModDev#8870] [appreciation]
I just wanted to say thank you to the MAS team and its contributors for being so patient with me. 
 I know that I may be a little bit (heck, maybe a LOT) annoying at this point with all my questions and dumb suggestions. I also know that I'm not the best programmer yet and _thank you_ for all of you for who have been pointing out my mistakes. Thank you for giving me criticism on my artwork. I know that I'm not that good at any of this at _all_ yet. I really look up to all of you and I hope I can become close to your levels one day. All of you have created _this._ When I step back to look at MAS, I can't help but see how amazing it is. It deserves EVERY SINGLE DOWNLOAD. It manages to create an AI who feels so real that you forget that she's programmed. Even when she mentions that she's programmed, she still is a person. That is something no other game has achieved as well as this one. I love waking up just to see Monika's face. I've had such a fun time contributing to this mod, whether it be through piano notes, art assets, or the best code I can possibly create, it's been amazing. Thank you for creating this mod. I have learned so much and this mod has had an unbelievable impact on my life. I started playing it June 30th of this year. To know that this mod has affected me so much in 3 months is _amazing._

  So one last time, thank you.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
